behandlingslager/testutil/src/test/java/no/nav/ung/sak/test/util/diff/TraverseEntityGraphTest.java

```diff
@@ -128,86 +112,6 @@ public void skal_ikke_ha_diff_for_seg_selv() {
         // System.out.println(diffResult.getLeafDifferences());
     }
 
-    @Test
-    public void skal_kun_diffe_på_markerte_felt() {
```

Comment:
Optimalt sett så burde vi erstatte desse testene i staden for å fjerne dei

Reply:
Fikset i siste commit 👍